### PR TITLE
Create index brefore wait for completion

### PR DESCRIPTION
**Pull request by xiaocai2333**
/kind improvement

### CODE REVIEW

File: pymilvus/client/grpc_handler.py [hunk 1]
```diff
@@ -854,6 +856,14 @@ def get_index_state(
...
         self, collection_name: str, index_name: str, timeout: Optional[float] = None, **kwargs
     ):
+        if kwargs.get("refresh", False):
+            index_info = self.describe_index(collection_name, index_name, timeout)
+            index_params = {
+                "index_type": index_info["index_type"],
+                "metric_type": index_info["metric_type"],
+                "params": index_info["params"],
+            }
+            self.create_index(collection_name, index_info["field_name"], index_params, timeout)
```
Comment (XuanYang-cn) [inline pymilvus/client/grpc_handler.py:866]: Do we also wait inside create_index? maybe use sync = False here

**Review by XuanYang-cn — CHANGES_REQUESTED** [review]

File: pymilvus/client/grpc_handler.py (hunk 1 above)
Comment (xiaocai2333) [reply pymilvus/client/grpc_handler.py:866]: yeah, you're right. I will fix it.

**Pull request merged**